src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/Signals.java

```diff
@@ -18,10 +18,15 @@
 final class Signals
 {
     public static final int CACHE_ENTRY_UPDATED_SIGNAL = 1;
-    public static final int CACHE_ENTRY_SIGNAL = 2;
     public static final int CACHE_ENTRY_ABORTED_SIGNAL = 3;
     public static final int REQUEST_EXPIRED_SIGNAL = 4;
     public static final int INITIATE_REQUEST_SIGNAL = 5;
+    public static final int REQUEST_RETRY_SIGNAL = 7;
+    public static final int CACHE_ENTRY_NOT_MODIFIED_SIGNAL = 8;
+    public static final int SERVE_CACHE_ENTRY_SIGNAL = 9;
+    public static final int REQUEST_ABORTED_SIGNAL = 10;
+    public static final int GROUP_REQUEST_RESET_SIGNAL = 11;
```

Comment:
Is this last signal still needed given that the `RESET` frames can be sent directly?

Reply:
This is to signal that the group request got reset.

Reply:
Would like to know if we can avoid the `SIGNAL` bypass and instead send the `RESET` frame directly.